manifests/s/Splashtop/SplashtopBusiness/3.5.4.0/Splashtop.SplashtopBusiness.locale.en-US.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Splashtop.SplashtopBusiness
+PackageVersion: 3.5.4.0

```

```suggestion
PackageVersion: 3.5.400.0
```
Must Match Control Panel